src/Paket.Core/InstallProcess.fs

```diff
@@ -169,6 +170,22 @@ let CreateModel(root, force, dependenciesFile:DependenciesFile, lockFile : LockF
     |> Seq.concat
     |> Seq.toArray
 
+type AssemblyDescriptor =
```

Comment:
please, check pull request for use more flex solution.

https://github.com/fsprojects/Paket/pull/1784